rpms/generate_rpm_package.sh

```diff
@@ -70,7 +70,8 @@ build_rpm() {
     DOCKERFILE_PATH="$2"
 
     # Copy the necessary files
-    cp ${CURRENT_PATH}/build.sh ${DOCKERFILE_PATH}
+    cp build.sh ${DOCKERFILE_PATH}
```

Comment:
Leave it as Debian.

Reply:
Done:[02a29f](https://github.com/wazuh/wazuh-packages/commit/02a29f8a7bdf3a69b9e485ca2f4d787337a8c06f)